jira-api/routes/api/v1/routes.js

```diff
@@ -79,31 +79,26 @@ app.get('/api/v1/getUsersForGroups/:groupName', asyncMiddleware(async function(r
     })
 }));
 
-app.get('/api/v1/getTimeForUsers/', asyncMiddleware(async function(req, res, next){
-
-    let group = req.query.groupName;
-    let startDate = req.query.startDate;
-    let endDate = req.query.endDate;
-    let boardId = req.query.boardId;
-    let sprintId = req.query.sprintId;
-
+app.get('/api/v1/getTimeForUsers/', asyncMiddleware(async (req, res, next) => {
+    let {
+        groupName, startDate, endDate, boardId, sprintId
+    } = req.query;
+    
     let groupMembers = [];
     //GET MEMBERS
     let members = await jiraservices.getUsersForGroup(group);
 
-    for(var member in members)
-    {
-        groupMembers.push({ key: members[member].key, worklogs: [], totalTimeInSeconds: 0 })
-    }
+    members.forEach(member => 
+        groupMembers.push({ key: members[member].key, worklogs: [], totalTimeInSeconds: 0 }));
 
     //GET ISSUES
     let jql = `worklogAuthor in membersOf("${group}") and worklogDate >= "${startDate}" and worklogDate <= "${endDate}"`;
     let issues = await agileservices.searchIssues(jql);
     let worklogs = [];
     let totalTimeForGroupSeconds = 0;
     let issuesTypeCounts = [];
-    for(var issue in issues)
```

Comment:
var in an iterator declaration is supper bad.  Always use let so it is bound to the block scope